pandas/_testing.py

```diff
@@ -82,6 +82,7 @@
 ALL_EA_INT_DTYPES = UNSIGNED_EA_INT_DTYPES + SIGNED_EA_INT_DTYPES
 
 FLOAT_DTYPES: List[Dtype] = [float, "float32", "float64"]
+FLOAT_EA_DTYPES: List[Dtype] = ["Float32", "Float64"]
```

Comment:
i think we also have a fixture in conftest and we have an aggregate of all ea types (can be added later but reallly good for testing now)

Reply:
There is indeed a `any_nullable_int_dtype` fixture in conftest.py. I could add an equivalent floating one, but right now it wouldn't be used yet. So can also add once we need it somewhere.

Reply:
kk but yes we should add one in this pr

Reply:
OK, added the fixture